### PR TITLE
Added #85 add login and register page

### PR DESCRIPTION
**Pull request by rohitgupta274509**
please review and approve it.

### CODE REVIEW

File: src/components/Pages/Login.vue [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      <router-link to="/sign-up">Sign Up</router-link>
+    </p>
+  </div>
+</template>
+
+<script>
+import InputText from "primevue/inputtext"
+import Password from "primevue/password"
+import Button from "primevue/button"
+import "primevue/resources/themes/lara-light-indigo/theme.css"
```
Comment (ankitvats1) [inline src/components/Pages/Login.vue:44]: We are not using this theme. Please update it.

Comment (ankitvats1) [reply src/components/Pages/Login.vue:44]: Also, add a comment somewhere that these imports will be removed in the future.

**Review by ankitvats1 — CHANGES_REQUESTED** [review]
Update the code with requested changes.

File: src/components/Pages/Pages.scss [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  margin: 0;
+}
+.logo {
+  width: 150px;
+  text-align: center;
+  -webkit-app-region: drag;
+}
+
+.sign-up {
+  font-family: Avenir, Helvetica, Arial, sans-serif;
```
Comment (ankitvats1) [inline src/components/Pages/Pages.scss:12]: We are not using these fonts

File: src/components/Pages/Pages.scss [hunk 3]
```diff
@@ -0,0 +1,81 @@
+body {
```
Comment (ankitvats1) [inline src/components/Pages/Pages.scss:1]: CSS for body must be in base scss file

File: src/components/Pages/Login.vue (hunk 1 above)
Comment (rohitgupta274509) [reply src/components/Pages/Login.vue]: while removing this theme file, facing same issue of styling. Comment added for these files.

File: src/components/Pages/Pages.scss (hunk 2 above)
Comment (rohitgupta274509) [reply src/components/Pages/Pages.scss]: removed these fonts

File: src/components/Pages/Pages.scss (hunk 3 above)
Comment (rohitgupta274509) [reply src/components/Pages/Pages.scss]: CSS remove from main file.

**Pull request merged**